cli/cmd/viewRunner.go

```diff
@@ -58,7 +58,9 @@ func runMizuView() {
 
 	logger.Log.Infof("Mizu is available at %s\n", url)
 
-	uiUtils.OpenBrowser(url)
+	if !config.Config.Tap.HeadlessMode {
```

Comment:
I think it should not be here.
(we are taking tap field in the view command) 
and when user runs the view command it make sense to always open browser (IMO).